test/steps/v2_steps.py

```diff
@@ -1740,6 +1788,155 @@ def build_payment_transaction(
     )
 
 
+def transient_acct_app_builder(
```

Comment:
Re-routing the various `I build an application transaction with the...` steps through this new method